setup.cfg

```diff
@@ -0,0 +1,64 @@
+;;
+;; Fonzie package
+;;
+[metadata]
+name = fun-apps
+version = 4.18.0
```

Comment:
if this has to be done manually we will forget :-(
any best practice to make sure it's updated ? 
Check in build when there is a tag that the tag is equal to this ?

Reply:
The standard way :tm: to make a new release is:

1. Create a new branch named `prepare-4.18.1`
2. Bump release number in `setup.cfg` (or `__init__.py`, depending on the way you handle your package version) and `package.json` (if any)
3. Update the `Changelog`
4. Commit your changes with a standard message title like: `Bump release to 4.18.1`
5. Open a pull request
6. Wait for validation
7. Tag & push your commit
8. Merge your PR

Hence code reviews must help no to forget. But. As we are only Human after all, we can enforce tag and version mapping in the CI, yes.

Reply:
Yes CI is here to help us stay coherent. Maybe your write-up above should go to the handbook WDYT?

Reply:
> Maybe your write-up above should go to the handbook WDYT?
You read my mind :stuck_out_tongue:

Reply:
I've added tag/version checking as follow: https://github.com/openfun/fun-apps/pull/629/files#diff-1d37e48f9ceff6d8030570cd36286a61R86

Reply:
FTR the software release procedure has been added to our handbook: https://openfun.gitbooks.io/handbook/content/git.html#releasing-new-software-version